ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -517,21 +518,17 @@ static void maybeCreateProcessingLogStream(
       final KsqlConfig ksqlConfig,
       final KsqlEngine ksqlEngine,
       final CommandQueue commandQueue) {
-    if (!config.getString(ProcessingLogConfig.STREAM_AUTO_CREATE)
-        .equals(ProcessingLogConfig.AUTO_CREATE_ON)) {
+    if (!config.getBoolean(ProcessingLogConfig.STREAM_AUTO_CREATE)) {
       return;
     }
-    final String name = config.getString(ProcessingLogConfig.STREAM_NAME);
-    final String topicName = config.getString(ProcessingLogConfig.TOPIC_NAME);
-    final String statementText = ProcessingLogServerUtils.processingLogStreamCreateStatement(
-        name,
-        topicName);
+    final PreparedStatement<AbstractStreamCreateStatement> statement =
```

Comment:
nit: it is worth moving this below the `if (!commandQueue.isEmpty()) {` check? i.e. no point constructing if we're not going to use...